txclib/commands.py

```diff
@@ -101,13 +101,14 @@ def cmd_config(argv, path_to_tx, is_legacy=False):
     """Add local or remote files under transifex"""
     from_wizard = False
     if len(argv) == 0:
-        # since interactive wizard should be equivalant to auto-local
+        # since interactive wizard should be equivalent to auto-local
         # subcommand there are some default options that need to be set
         default_options = {
             'execute': True,
             'subcommand': MAPPING,
             'minimum_perc': 0,
             'mode': None,
+            'expression': 'locale/<lang>/{filepath}/{filename}{extension}',
```

Comment:
Here you need to set `expression_legacy`, `expression` is already set by the Wizard class. And it can simple be `expression_legacy: None`

Reply:
@SofiaMargariti ACK